app/templates/direct-award/did-you-award-contract.html

```diff
@@ -1,7 +1,7 @@
 {% extends "_base_page.html" %}
 
 {% block pageTitle %}
-  Did you award a contract? - Digital Marketplace
+  {% if errors %}Error: {% endif %}Did you award a contract? - Digital Marketplace
```

Comment:
Rather than doing this for all pages, can we do this once in the base page template?

Reply:
The way it's currently set up, I think we'd have to use:

```
{% set page_name = "Save your search" %}

{% block pageTitle %}
  {{ super() }}{{ page_name }} - Digital Marketplace
{% endblock %}```

Which is similar in terms of "update all the pages"-iness, but not quite the same as just setting an error message (in case there's something else in the parent block).

So I think our options look like:

1. Update each page with the super() call.
2. Generate page names in the presenters.
3. Use a global page title variable and a function to set it
4. Add the `if errors` to each page

Reply:
What about

```
# base_page.html
{% block pageTitle %}
  {% if errors %}Error: {% endif %}{{ page_name}} - Digital Marketplace
{% endblock %}
```

Would we need the super then?

Reply:
Passing from the child to the parent through blocks doesn't seem to work.

https://stackoverflow.com/questions/24833370/pass-variables-from-child-template-to-parent-in-jinja2/24836460#24836460

Reply:
Oh wait, and not have the block in the child? I'll give that a go.

Reply:
It works!

Reply:
It that doesn't work, using a block for pageName should.

```
# base_page.html
{% block pageTitle %}
  {% if errors %}Error: {% endif %}{% block pageName %}{% endblock %} - Digital Marketplace
{% endblock %}

# child
{% block pageName%}Save your search{% endblock %}
```

Reply:
Changes made. Running functional tests locally now.